bosh/opsfiles/production.yml

```diff
@@ -0,0 +1,75 @@
+- type: replace
+  path: /name?
+  value: shibboleth-production
+
+- type: replace
+  path: /instance_groups/name=idp/properties/idp?
```

Comment:
This will overwrite all the properties that are already defined at this path in the base manifest. Maybe use multiple smaller operations?

Reply:
I looked at the manifest, and there were no other idp properties, so I thought this would not be a problem.  But I'm happy to break them out into individual operations.

Reply:
Bah.  I missed that the properties here are actually under jobs, so this is all wrong.  Will have it fixed up soon.